TrustlyAndroidSdk/Trustly-Android-Library/build.gradle

```diff
@@ -1,19 +1,23 @@
+apply plugin: 'com.android.library'
 
-
-
-
-buildscript {
-    repositories {
-        google()
-        jcenter()
-    }
-}
-
-plugins {
-    id 'maven-publish'
+ext {
+    // Sonatype coordinates
```

Comment:
Since this is for the public - should it actually say `Sonartype`?

Reply:
This are publicly available through Sonatype as well, so nothing secret is disclosed here. But I can remove the comment if you think it's unnecessary